azure-functions/acbs-function/src/functions/acbs-amend-facility.js

```diff
@@ -77,21 +89,27 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
 
     if (!acceptableFacilityStage.includes(facilityStageCode)) {
       // Error upon unacceptable facility stage
-      const incorrectFacilityStageErrorMessage = `Facility ${facilityId} stage is ${facilityStageCode}, amendment will not be processed`;
+      const incorrectFacilityStageErrorMessage = `Facility ${facilityId} stage is ${facilityStageCode}, amendments are accepted for 07 stage.`;
```

Comment:
What is the meanint of "07" Seems to be an ACBS specific facility stage - wondering if it's worth adding to line 85 what this actually means rather than an ambiguous number